content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -79,7 +79,35 @@ $ dvc exp run --all-pipelines
 > Note that the order in which pipelines are executed is not guaranteed; Only
 > the internal order of stage execution is.
 
-### Interactive reproduction
+### Running pipelines recursively
+
+DVC supports pipelines defined in more than one `dvc.yaml` file. These can
+reside in nested folders and you may want to run all these in a single command.
+
+```
+$ tree
+.
+├── dir
+│   ├── deeper-dir
+│   │   └── dvc.yaml
+│   ├── dvc.lock
+│   └── dvc.yaml
+├── dvc.lock
+└── dvc.yaml
+```
+
+In this example, if you want to run the pipelines defined in `dir/dvc.yaml` and
+`dir/deeper-dir/dvc.yaml` in a single command, you can issue:
+
+```dvc
+$ dvc exp run --recursive dir/
+```
+
+It will run all the pipelines under `dir/`. You may also use `.` as a directory,
+so `dvc exp run -R .` to run all the pipelines recursively, starting from the
+current directory.
```

Comment:
```suggestion
It will run all the pipelines under `dir/`.

You may also use `.` as target (e.g.
`dvc exp run -R .`) to run all pipelines in the entire project
(recursively starting from the current directory).
```

Reply:
(Committing this last one copy edit)